docs/source/deployment/lambda.mdx

```diff
@@ -0,0 +1,387 @@
+
+<!-- cSpell:disable-next-line -->
+
+To find the S3 bucket that Serverless created, search in Amazon's listed services for S3, then look for the name of your bucket (e.g., `apollo-lambda-dev-serverlessdeploymentbucket-1s10e00wvoe5f` is the name of our bucket).
+
+To find the Lambda function that Serverless created, search in Amazon's listed services for `Lambda`. Double-check the region at the top right of the screen if your list of Lambda functions is empty or missing your new function. The default region for Serverless deployments is `us-east-1` (N. Virginia).
+
+If you ever want to remove the S3 bucket or Lambda functions that Serverless created, you can run the following command:
+
+```bash
+ serverless remove
+```
+
+## Customizing HTTP behavior
+
+The `@as-integrations/aws-lambda` package is compatible with Lambda's API Gateway V1 (REST) and V2 (HTTP). However, it does not let you customize HTTP behavior directly.
+
+If you'd like to customize your HTTP behavior, you can couple Apollo Server's Express integration (i.e., [`expressMiddleware`](../api/express-middleware)) with the [`@vendia/serverless-express`](https://github.com/vendia/serverless-express) package. The `@vendia/serverless-express` library translates between Lambda events and Express requests.
+
+You can update your Apollo Server setup to the following to have a fully functioning Lambda server that works in a variety of AWS features:
+
+<MultiCodeBlock>
+
+```ts
+const { ApolloServer } = require('@apollo/server');
+const { expressMiddleware } = require('@apollo/server/express4');
+const serverlessExpress = require('@vendia/serverless-express');
+const express = require('express');
+const { json } = require('body-parser');
+const cors = require('cors');
+
+const server = new ApolloServer({
+  typeDefs: 'type Query { x: ID }',
+  resolvers: { Query: { x: () => 'hi!' } },
+});
+
+server.startInBackgroundHandlingStartupErrorsByLoggingAndFailingAllRequests();
+
+const app = express();
+app.use(cors(), json(), expressMiddleware(server));
+
+exports.graphqlHandler = serverlessExpress({ app });
+```
+
+</MultiCodeBlock>
+
+The setup enables you to customize your HTTP behavior as needed.
+
+## Using request information
+
+You can use the [`context` function](../data/resolvers/#the-context-argument) to get information about the current operation from the original Lambda data structures. Your `context` function can access this information from the passed-in `event` and `context` arguments:
+
+<MultiCodeBlock>
+
+```ts
+const server = new ApolloServer<MyContext>({
+  typeDefs,
+  resolvers,
+});
+
+// This final export is important!
+export const graphqlHandler = startServerAndCreateLambdaHandler(server, {
+  // highlight-start
+  context: async (event, context) => {
```

Comment:
I believe this takes `event` and `context` as options on a single argument.

Reply:
Oh, thank you! Great catch 🔥